File: app/src/main/java/com/BlizzardArmory/warcraft/Talents/Key.java
package com.BlizzardArmory.warcraft.Talents;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Key {

    @SerializedName("href")
    @Expose
    private String href;

    public String getHref() {
        return href;
    }

    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/account/Key.kt
package com.BlizzardArmory.model.warcraft.account

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * The type Key.
 */
data class Key(

        @SerializedName("href")
        @Expose
        var href: String

)